docs/antrea-l7-network-policy.md

```diff
@@ -362,6 +364,34 @@ Deny ingress from client (10.10.1.5) to web (10.10.1.4/admin)
 }
 ```
 
+### L7 Visibility
+
+Layer 7 network flow visibility enables users to see the L7 flows in the network and
+visualizing it on Theia.
+
+#### Usage
+
+To achieve L7 (Layer 7) network flow visibility, feature gates,
+namely `L7Visibility` and `TrafficControl`, must be enabled.
+User could utilize the L7 network visibility feature either by applying l7 network policies
+as mentioned above or by annotating the pods or namespaces.
+
+we have introduced `l7visibility` as the annotation key which supports network visibility
+of the L7 networks on a pod or namespace for ingress/egress/both directions.
+
+To annotate a pod(`test-pod`) on default namespace and using flow direction `both`
+
+```bash
+kubectl annotate pod test-pod l7visibility=both
+```
+
+Based of the annotation or the networkpolicy, flow exporter would export the flows to the
+clickhouse or s3uploader using the fields `isL7` and `httpvals`
+
+`httpvals` would be of the type (`Hostname<>URL<>UserAgent<>ContentType<>Method<>Protocol<>Status<>ContentLength`)
```

Comment:
Is this a standard format? If not, why not use something more structured like JSON?

Reply:
This is not a standard format, I shall be looking into a more standized format. Thanks for the suggestion.